### PR TITLE
update to Node 8 and flexible csvStringify

### PR DESCRIPTION
**Pull request by oprogramador**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "eslint-config-airbnb-improved"
```
Comment (crookedneighbor) [inline .eslintrc:2]: I don't see the need to switch from standard to eslint.

File: README.md [hunk 2]
```diff
@@ -37,25 +37,33 @@ stringify(input, options).then((output) => {
...
 
 stringify(input).then((output) => {
   // uses bluebird instead of Promise
 }).catch((err) => {
   // handle error
-})
+});
+```
+
+### Use your own csv-stringify library
```
Comment (crookedneighbor) [inline README.md:54]: This is an interesting idea, but basically requires that the custom csv library follows the exact same shape with the exact same options as csv-stringify, so I don't see that this is ultimately a useful feature.

File: package.json [hunk 3]
```diff
@@ -1,10 +1,11 @@
 {
   "name": "csv-stringify-as-promised",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "csv-stringify wrapped in a promise",
   "main": "lib/index.js",
   "scripts": {
-    "lint": "standard | snazzy",
+    "lint": "./node_modules/.bin/eslint lib test",
+    "test-mocha": "./node_modules/.bin/mocha",
```
Comment (crookedneighbor) [inline package.json:8]: You don't need to specify `./node_modules/.bin` in npm scripts. npm automatically looks for those binaries when doing `npm run ...`

**Review by crookedneighbor — COMMENTED** [review]
I don't think there's a compelling reason to make these additional changes.

I would be open to reviewing a PR that updates the version of `csv-stringify` that is used mark node support as v7 and higher (the version that library supports).

File: package.json (hunk 3 above)
Comment (oprogramador) [reply package.json:8]: but it can use the global dependencies which may be in other versions

File: README.md (hunk 2 above)
Comment (oprogramador) [reply README.md:54]: ex. I can fork `csv-stringify` which will have the same interface

Comment (crookedneighbor) [reply README.md:54]: I think the logic around this is pretty simple. If someone is forking csv-stringify already, they can just include a Promise implimentation and there's no need to use this module. 

That said, I no longer use this module and would be happy to transfer ownership to a new maintainer. So if you want to maintain it and make these changes, go for it. Just let me know what your npm username is.

Comment (oprogramador) [reply README.md:54]: my NPM username is the same as the GitHub username - `oprogramador`

Comment (crookedneighbor) [reply README.md:54]: Done.

**Pull request closed without merging**